source/extensions/omni.isaac.lab/omni/isaac/lab/assets/articulation/articulation_data.py

```diff
@@ -70,6 +70,9 @@ def __init__(self, root_physx_view: physx.ArticulationView, device: str):
         self._joint_acc = TimestampedBuffer()
         self._joint_vel = TimestampedBuffer()
 
+        # Link center of mass
+        self._com_pos_b, _ = self._root_physx_view.get_coms().to(self.device).split([3, 4], dim=-1)
```

Comment:
What if user sets the "com" for randomization. This would affect this property, right?

Reply:
Yeah that was one hangup I ran into with testing. One way to get around this is to move this to a property that others call. I was trying to limit the times it gets pulled from the root_physx_view, but with domain randomization I think it is inevitable.